src/assessments/semantics/test-steps/headers.tsx

```diff
@@ -70,7 +70,7 @@ const key = SemanticsTestStep.headers;
 
 export const Headers: Requirement = {
     key,
-    name: 'Headers',
+    name: 'Row/Column headers',
```

Comment:
Is capitalizing the "C" in "Column" correct here? I actually don't know the style requirement